zerver/tests/test_user_topics.py

```diff
@@ -36,7 +36,7 @@ def test_get_deactivated_muted_topic(self) -> None:
             self.assert_json_success(result)
 
         stream.deactivated = True
-        stream.save()
+        stream.save(update_fields=["deactivated"])
```

Comment:
Should this be using `do_deactivate_stream` instead? Usually we aim to use real code like that in tests.

Reply:
In general, we've been trying to clean up tests to use action functions for creating state, so you could consider replacing the test parts of this commit with a prep PR that does that -- I think it's just the `date_created` ones that would still do hacky edits.

Reply:
ok